.vscode/settings.json

```
@@ -0,0 +1,3 @@
+{
+    "cmake.sourceDirectory": "C:/Users/ritot/OneDrive/デスクトップ/高度ICT/funverse_app/linux"
```

絶対パスだと他の環境で対応できないので、相対パスで指定してほしい

```suggestion
    "cmake.sourceDirectory": "./linux"
```

（とはいえlinux向けビルドはいらないのでディレクトリから消しちゃっていい気もするが...）

---

やりました